k8s-test.bash

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -xe
+
+TEST_NAMESPACE="${TEST_NAMESPACE:-k8s-test}"
+RELEASE_NAME="${RELEASE_NAME:-cryostat-k8s-test}"
+
+if [ "${CREATE_CLUSTER:-true}" = "true" ]; then
+    kind create cluster
+fi
+
+kubectl create ns "${TEST_NAMESPACE}"
+function cleanup() {
+    kubectl delete ns "${TEST_NAMESPACE}"
+    if [ "${CREATE_CLUSTER:-true}" = "true" ]; then
+        kind delete cluster
+    fi
+}
+trap cleanup EXIT
```

Comment:
Maybe this part can come before setting up the cluster? Just in case some error in the setup that won't be cleaned up.